components/formats-gpl/src/loci/formats/in/MetamorphReader.java

```diff
@@ -937,7 +936,7 @@ else if (exposureTimes.size() == 1 && exposureTimes.size() < getSizeC()) {
 
       for (int p=0; p<getImageCount(); p++) {
         int[] coords = getZCTCoords(p);
-        Double deltaT = Double.valueOf(0);
+        Double deltaT = 0d;
```

Comment:
Probably just one I happened to notice, I'm progressively getting them over time!